src/Microsoft.AspNetCore.Sockets.Client/LongPollingTransport.cs

```diff
@@ -132,16 +134,19 @@ public Task StartAsync(Uri url, IChannelConnection<Message> application)
 
                         var response = await _httpClient.SendAsync(request);
                         response.EnsureSuccessStatusCode();
+                        sendTcs.SetResult(null);
```

Comment:
Dispatch the TCS completion. What version of .NET Standard are we using? We may be able to use the new TCS enum.